src/spikeinterface/core/job_tools.py

```diff
@@ -206,9 +196,11 @@ def ensure_chunk_size(
     chunk_size: int or None
         size for one chunk per job
     chunk_memory: str or None
-        must end with "k", "M" or "G"
+        must end with "k", "M", "G", etc for decimal units and "ki", "Mi", "Gi", etc for
+        binary units. (e.g. "1k", "500M", "2G", "1ki", "500Mi", "2Gi")
```

Comment:
@h-mayorquin I think that this might make things extra complicated. If I understand correctly, 

1KB = 1000 bytes
1KiB = 1024 bytes

Is that correct? I had to look up the difference between decimal/binary for memory representation, so I think that an end user could be a bit puzzled by this. 

Happy to discuss!

Reply:
I think that having the two is confusing indeed.

But the current setup is confusing as well. If somebody allocates 1G using the current SpikeInterface machinery  (and SpikeInterface was perfect in its handling of memory ...) the memory reported by the system won't match 1G as the user specified.

The reason for this is that RAM memory is reported in binary base. Both when you buy RAM and when you use utilities like system resource, the numbers you see are in binary base. .

Given that memory allocation here is a RAM issue, I think we should follow the convention of RAM specification (binary). However, we have to do things carefully because the decimal base has been there for so long.

Finally, given that using the job_kwargs is a relatively advanced feature, I am far less concerned about this type of confusion and I am fine with having the user specify which one they want to use.

(Hard drive storage, on the other hand, is reported in base 10, both when you calculate the size of an object in the system and when you buy a hard drive)